recipes/empiricaldist/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "empiricaldist" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/empiricaldist-{{ version }}.tar.gz
+  sha256: f7b07f5877bf5317a763e84fb5e13e06ffda66aca49e35a857d793aae68dd052
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib-base
+    - numpy
+    - pandas
+    - seaborn
```

Comment:
These are not listed upstream but I can see that they are being used. Would be good to have them upstream so that pip check can also work correctly.

Reply:
The dependencies are now explicitly listed upstream

https://github.com/AllenDowney/empiricaldist/blob/1e68b0023ce58943ad4a3324820eb183e61a628e/setup.py#L18

matplotlib and seaborn aren't true dependencies. They are used in some example jupyter notebooks, but these are not distributed in the package tarball

Reply:
The dependencies are now explicitly listed upstream

https://github.com/AllenDowney/empiricaldist/blob/1e68b0023ce58943ad4a3324820eb183e61a628e/setup.py#L18

matplotlib and seaborn aren't true dependencies. They are used in some example jupyter notebooks, but these are not distributed in the package tarball

Reply:
For future reference: turns out matplotlib was a true dependency. We subsequently fixed the issue upstream and in the feedstock

https://github.com/AllenDowney/empiricaldist/pull/24
https://github.com/conda-forge/empiricaldist-feedstock/pull/1#issuecomment-1120085591